src/main/com/leandro/it/FibonacciFunction.java

```diff
@@ -0,0 +1,25 @@
+package com.leandro.it;
+
+import exceptions.InvalidNumberException;
+
+public class FibonacciFunction {
+
+    int n;
+
+    public FibonacciFunction() {
+
+    }
+
+    public int fibonacciN(int n) {
```

Comment:
could you practice renaming this function? i think that if I review your code in 2 years y couldn't understand what make this through this name